datasets/bbc_hindi_nli/README.md

```diff
@@ -0,0 +1,181 @@
+---
+annotations_creators:
+- machine-generated
+language_creators:
+- found
+languages:
+- hi
+licenses:
+- mit
+multilinguality:
+- monolingual
+size_categories:
+- 10K<n<100K
+source_datasets:
+- extended|bbc__hindi_nli
```

Comment:
Here we expect to list the datasets that were used to construct this one.
As far as I understand this dataset contains original data from news:
```suggestion
- original
```

Reply:
Not really so we have created the  NLI dataset from [BBC Hindi News Classification ](https://www.kaggle.com/c/learn-ai-bbc) So thats why I have used the extended tag. I have modified the dataset name accordingly.